app/views/adopters/_adopter_form.html.erb

```diff
@@ -5,12 +5,17 @@
         <% if @adopter.adoption_app.has_family_under_18 == true %>
           <span class="label label-warning">Has Family Under 18</span>
         <% end %>
+
         <% if @adopter.adoption_app.is_ofage == false %>
           <span class="label label-warning">Under 21</span>
         <% end %>
 
-        <% if @adopter.adoption_app.pets_branch == "no_pets" %>
+        <% if @adopter.adoption_app.pets_branch == "no_pets" && @adopter.adoption_app.prev_pets_type.empty? %>
```

Comment:
You'll want to use `.blank?` here instead of `.empty?` as any existing applications in the system will have a NULL `prev_pets_type`, and it will throw
```
ActionView::Template::Error (undefined method `empty?' for nil:NilClass)
```

https://blog.arkency.com/2017/07/nil-empty-blank-ruby-rails-difference/

Reply:
Thanks Mark!